templates/service-template.yml

```diff
@@ -590,20 +590,6 @@ objects:
     data:
       kafka-sre-user-list.yaml: |-
         ${KAFKA_SRE_USERS}
-  - kind: ConfigMap
-    apiVersion: v1
-    metadata:
-      name: kas-fleet-manager-kafka-capacity-config
-      annotations:
-        qontract.recycle: "true"
-    data:
-      kafka-capacity-config.yaml: |-
-        ingressEgressThroughputPerSec: ${KAFKA_CAPACITY_INGRESS_THROUGHPUT}
-        totalMaxConnections: ${KAFKA_CAPACITY_TOTAL_MAX_CONNECTIONS}
-        maxDataRetentionSize: ${KAFKA_CAPACITY_MAX_DATA_RETENTION_SIZE}
-        maxPartitions: ${KAFKA_CAPACITY_MAX_PARTITIONS}
-        maxDataRetentionPeriod: ${KAFKA_CAPACITY_MAX_DATA_RETENTION_PERIOD}
-        maxConnectionAttemptsPerSec: ${KAFKA_CAPACITY_MAX_CONNECTION_ATTEMPTS_PER_SEC}
```

Comment:
Do we want to remove the `KAFKA_CAPACITY_XX` https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/2bbd6264ea829806205681c7359a5f00a24934a3/templates/service-template.yml#L421-L449 parameters as well?